src/reducers/platform.js

```diff
@@ -50,7 +62,13 @@ export function api(
 				return { ...DEFAULT_API_STATE, inFlight };
 			}
 
-			return { ...state, inFlight };
+			// remove any `ref`s that are being refreshed - eliminate stale data
```

Comment:
Are we also clearing inFlight when errors occur, i _think_ i observed that not to be the case